Phoron is now orange, across the board.

chaoko99 opened the pull request:
This was a less annoying solution, I admit.

Screenshots. (I may have missed some stuff. You should be able to get the gist, they're all tinted the same color)

![image](https://cloud.githubusercontent.com/assets/4706052/24078790/bc306638-0c34-11e7-9039-3078f0185722.png)

chaoko99 commented:
Please let me know if I missed any. I SWEAR I missed one or two.

SiegDerMaus commented:
I'd rather have everything purple than everything orange, to be honest.

Kasuobes commented:
Going from grape juice to orange juice, bleh.

Actually I'm fine with it. The uglier you make them the less likely people will be silly enough to make furniture out of phoron.

Or maybe I'm just being hopeful.

chaoko99 commented:
@Kasuobes If we wanna go full ham on that, I can always uncomment that thing that makes phoron furniture explode when exposed to an open flame

Kasuobes commented:
Please do. In another PR though, maybe, because people can't seem to handle combo platters and instead prefer à la carte.

chaoko99 commented:
Done. It seems to only care about the flashpoints though. I might have to look into the exposure to lighters and whatnot.

You bring out the worst in me, Kasuobes — Thank you for that. Spite is one of those things that helps me get things done.

Albens commented:
I like this.
It's not grape-flavoured phoron. It's instead like.. solid fire.

+1 from me, but it means nothing.

PsiOmegaDelta commented:
I suppose this is nice, but not 17 commits nice.

chaoko99 commented:
I made the commits as I made the stuff itself (I have this terrible habit of losing raws.).  I can stuff it all into one commit, if it really is that big of an issue.

NortonDK commented:
wait, you can have phoron power cells?

chaoko99 commented:
No, those are just admin spawned infinite cells.
They were for the spotlights.. I forgot to map in a screwdriver.

chaoko99 committed:
Phoron chemical colors


Phoron material colors


Orange flavored phoron sheets

Do these have an inhand? I can't find one.
>Implying phoron pickaxes justify their own existence in the code

Left the old sprite in because hey, it's an icon state.
MEATY ORE


apparently the Ppickaxe has inhands.

eh. Let's fix 'em too.
protolathe loading anim


Ore and wall ore changes


Orange phoron crystal.

I don't know what posessed me to make this.
Adds a fifty sheet phoron stack to DM

Why was this an issue
phorondoors

send help
Changelog


adds an icon state to the legacy crystal

 I don't know why I made this sprite, or even why I expect it to even see the light of day, but here we are.
changelog

how does this keep happening

D O I N ' I T


DID IT


missed a quotation mark

chaoko99 commented:
Likely going ot close this and reopen with all the commits in one.... I really hope this isn't a bother.

Edit: Figured it out.

Techhead0 commented on code/modules/xenoarcheaology/finds/misc.dm:
```diff
@@ -10,6 +10,9 @@
 		icon_state = "crystal2"
+	else
+		if(prob(50))
+			icon_state = "crystal3"
```
This `New()` should call `..()`. I know it's not your fault, but I'd appreciate it if it was fixed.

Also, the probabilities are unbalanced, if you care.

Also also, the formatting, although functional, is a little clunky.

chaoko99 replied:
Done.

chaoko99 committed:
Edits legacy crystal code

also makes grinders accept fiftysheets

chaoko99 committed:
Makes mappable crystals their own variant.

Also readds the old ppick inhands. They are ppickaxe-old

chaoko99 commented:
I still can't believe this got merged.

chaoko99 commented:
I still can't believe that this got merged. That phoron sprite looks awful.
